### PR TITLE
(DOCSP-9185): Port Execute a GraphQL Operation

### PR DESCRIPTION
**Pull request by nathan-contino-mongo**
JIRA: https://jira.mongodb.org/browse/DOCSP-9185

Stage: https://docs-mongodbcom-staging.corp.mongodb.com/realm/pegasus/DOCSP-9185/graphql/execute.html

### CODE REVIEW

File: source/graphql/execute.txt [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+             </div>
+           );
+         }
+   
+   .. tab:: graphqurl (CLI)
+      :tabid: graphqurl
+
+      .. code-block:: shell
+         
+         gq https://stitch.mongodb.com/api/client/v2.0/app/<Your App ID>/graphql \
```
Comment (nlarew) [inline source/graphql/execute.txt]: We'll need to follow up on these URLs

File: source/graphql/execute.txt [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+Run a Query
+-----------
+
+.. tabs::
+
+   .. tab:: Apollo (React)
+      :tabid: apollo-react
+      
+      Define the query with the ``qgl`` tag and then use the
+      :apollo-docs:`useQuery() <react/api/react-hooks/#usequery>` hook
```
Comment (nlarew) [inline source/graphql/execute.txt]: This isn't rendering, let's add `apollo-docs` to `conf.py`

File: source/graphql/execute.txt (hunk 1 above)
Comment (nathan-contino-mongo) [reply source/graphql/execute.txt]: Agreed. In another PR, @cbush mentioned that we can probably just leave them alone for now -- that's what I'll do here as well, I think.

File: source/graphql/execute.txt (hunk 2 above)
Comment (nathan-contino-mongo) [reply source/graphql/execute.txt]: Added, and created a PR to add this to snooty as well.

**Pull request merged**